class Calculations:

    def addition(x,y):
        try:
            return x+y
        except ValueError as exception:
            return exception
